molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/admin/site_languages_summary.html

```diff
@@ -0,0 +1,13 @@
+{% raw %}
+{% load i18n wagtailadmin_tags %}
+
+{% for summary in summaries %}
+  <li class="icon icon-doc-empty-inverse">
+    {% if summary.total > 1 %}
+      <span>{{ summary.total }}</span>{{ summary.language }} Pages
```

Comment:
You should use [Django's pluralize filter](https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#pluralize)